requirements.txt

```diff
@@ -1 +1,2 @@
 ops >= 1.5.0
+kubernetes
```

Comment:
Would it be worth pinning the version here? Maybe to the current major version?

Reply:
I set a lower bound for Kubernetes version, I would love to use the latest Kubernetes version to be future proof.

Reply:
Something to consider is that if, for example, `kubernetes` releases version `26` of the library with hypothetically breaking changes that matter for this charm, will that potentially break new deployments of the charm? I'm not sure if the install happens before publishing to charmhub or after, if after then it could be problematic. For Our dev tooling, not specifying a version is good since it just forces is to upgrade, on production dependencies it depends. For machine charms, for example, it might be problematic

Reply:
Update the requirement to `kubernetes ~= 25.3.0`